hydra-node/test/Hydra/Chain/Direct/Contract/Contest.hs

```diff
@@ -181,6 +194,13 @@ data ContestMutation
     MutateContesters
   | -- | See spec: 5.5. rule 6 -> value is preserved
     MutateValueInOutput
+  | -- | Change the 'ContestationDeadline' in the 'Closed' output datum such that deadline is pushed away
+    NotUpdateDeadlineAlthoughItShould
+  | -- | Pushes the deadline although this is the last contest. Instead of
+    -- creating another healthy case and mutate that one, this mutation just
+    -- changes the starting situation so that everyone else already contested.
+    -- Remember the 'healthyContestTx' is already pushing out the deadline.
+    PushDeadlineAlthoughItShouldNot
```

Comment:
Must: add a mutation which changes the `contestationPeriod` in the output datum. 

Any change is not allowed.